config/mclag.py

```diff
@@ -0,0 +1,348 @@
+#!/usr/sbin/env python
```

Comment:
do we need this ?

Reply:
it is not required. but it shouldn't harm.if required i can remove it

Reply:
@arlakshm i have adddressed the comment and resubmitted